lib/src/open_food_api_client.dart

```diff
@@ -1131,10 +1143,12 @@ class OpenFoodAPIClient {
       queryType: queryType,
       addCredentialsToBody: false,
     );
-    return json.decode(response.body);
+    return HttpHelper().jsonDecode(response.body);
   }
 
   /// cf. https://openfoodfacts.github.io/openfoodfacts-server/reference/api-v3/#get-/api/v3/taxonomy_suggestions
+  ///
+  /// Consider using [SuggestionManager].
```

Comment:
Could you update the README

Reply:
> Could you update the README

I will once the code is merged, as I cannot point to the doc of a class that does not exist yet.